dashboard/src/components/Atoms/Config/InputBox.tsx

```diff
@@ -43,6 +44,8 @@ export default function InputBox({
   isLoading: isLoadingProp = false,
   id = '',
   showIcons = true,
+  tooltip = false,
+  tooltipText,
```

Comment:
might be nice to use just 1 prop for tooltip,
since we want to enable tooltip if and only if we have tooltip text